### DISCUSSION TITLE
CPU speed on the RG35XX

### DISCUSSION THREAD

**Discussion by MancV21** [rg35xx-cfw/Koriki#76] [Q&A]
Hi. As far as I know the RG35XX has a clock speed of 1.6mhz, yet Koriki only allows me to 'overclock' to 1296mhz in the settings. Is this an oversight or done for a reason? Is there a way to manually set the CPU to 1.6 in the files at all?

Cheers.

**Comment by acmeplus** [comment]
It's on purpose, the addition of the GPU makes the system less permissive at higher overclock values. You can try, but in most of the tests, ~1.3GHz was about the max stable frequency, beyond that you will quickly experience green screens that are basically system crashes. 

The values of overclocking are defined in the code but you can manually override that by changing the value of ``/sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq``, e.g. with a script in roms/ports that does ``echo 1392000 > /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq``

With each overclock increment the system increases about 50mV that impacts in other areas like the GPU consumption. 1296MHz is considered the stable frequency.

**Comment by job1022** [comment]
I don't see any changes on a game after i go into settings and overclock the system. It always go back to the 1000mhz.

**Reply by ssangster** [reply]
I have same issue, any solution?

**Reply by Res777** [reply]
Unfortunately I'm also thinking that the project has been abandoned... no way to save the freq to 1296mhz

**Reply by acmeplus** [reply]
No. Not it’s not abandoned. All that is fixed in an upcoming 1.0.4 version but it’s not finished yet. There were regressions introduced that require fixes.
